spk/pyload/src/dsm-control.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+# Package
+PACKAGE="pyload"
+DNAME="pyLoad"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PYTHON_DIR="/usr/local/python"
+PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="pyload"
+PYTHON="${INSTALL_DIR}/env/bin/python"
+PYLOAD="${INSTALL_DIR}/share/pyload/pyLoadCore.py"
+LOG_FILE="${INSTALL_DIR}/etc/Logs/log.txt"
+PID_FILE="${INSTALL_DIR}/var/pyload.pid"
+
+
+
+start_daemon ()
+{
+    su ${USER} -c "PATH=${PATH} ${PYTHON} ${PYLOAD} --pidfile=${PID_FILE} --daemon"
+}
+
+stop_daemon ()
+{
+    su ${USER} -c "PATH=${PATH} ${PYTHON} ${PYLOAD} --pidfile=${PID_FILE} --quit"
+}
+
+daemon_status ()
+{
+    su ${USER} -c "PATH=${PATH} ${PYTHON} ${PYLOAD} --pidfile=${PID_FILE} --status > /dev/null"
```

Comment:
I think you want this:
```
su ${USER} -c "PATH=${PATH} ${PYTHON} ${PYLOAD} --pidfile=${PID_FILE} --status" > /dev/null
```
instead of this:
```
su ${USER} -c "PATH=${PATH} ${PYTHON} ${PYLOAD} --pidfile=${PID_FILE} --status > /dev/null"
```